frontend/app/src/services/rotkehlchen-api.ts

```diff
@@ -560,21 +560,23 @@ export class RotkehlchenApi {
       .then(handleResponse);
   }
 
-  importDataFrom(
+  async importDataFrom(
     source: string,
     file: string,
     timestampFormat: string | null
-  ): Promise<boolean> {
-    return this.axios
-      .put<ActionResult<boolean>>(
+  ): Promise<PendingTask> {
```

Comment:
nit: since we started changing it in other parts would it be possible to replace the `.then()` with the awaiting we do in other parts?